src/makeRobotAccountant.js

```diff
@@ -7,6 +7,21 @@
 
 function makeRobotAccountant() {
   // write code here
+  let callCounter = 0;
+
+  const getSum = (num) => {
+    callCounter++;
+
+    return (operator) => {
```

Comment:
[CODE STYLE] - You don't need to name functions if you return them right away. In this case, the function returned from `getSum` is anonymous as it should be, which aligns with the code style guide.